doc/dev/corpus.rst

```diff
@@ -55,11 +49,6 @@ Do some more stuff with rgw if you know how.
 
 	../src/stop.sh
 
-#. Import the corpus (this will take a few minutes)::
```

Comment:
@iotcg why are you removing the section for importing corpus? i think it's an important step for preparing the baseline of corpus.

Reply:
also, since you are here. could you please update the doc referencing

```
make check-local
```

Reply:
@tchaikov : do_autogen.sh has been deleted by commit: https://github.com/ceph/ceph/commit/ea9b972114fb3a14657806d227015e45aa3caea0

Reply:
yes, i think there is another option, to use `do_cmake.sh -DCMAKE_CXX_FLAGS="-DENCODE_DUMP_PATH=/tmp/foo"` instead of removing this section.

Reply:
@tchaikov Could you help update doc/dev/corpus.rst to do the change? I'm not familiar with ceph/corpus.

Reply:
sure. please keep this section. and i will revise it.

Reply:
#27552